docs/howitworks.md

```diff
@@ -263,6 +263,18 @@ Valid examples:
 
 This can be used for both generic and inline-path placeholders.
 
+##### `base64decode`
+
+The base64decode modifier decodes base64 encoded values into plain-text.
+
+Valid examples:
+
+- `<b64_username | base64decode>`
+
+- `<path:secrets/data/my-db#b64_username | base64encode>`
```

Comment:
I think you mean `decode` here :)

Reply:
Feel free to fix here, code looks great otherwise so I left an approval

Reply:
Great catch! Just updated.